polylogue/config.py

```
@@ -132,13 +132,19 @@ def is_config_declarative(path: Optional[Path] = None) -> Tuple[bool, str, Path]
             return True, "config file is read-only", target
     else:
         parent = target.parent
+        existing_parent = parent
         try:
-            parent_resolved = parent.resolve()
+            while not existing_parent.exists() and existing_parent != existing_parent.parent:
+                existing_parent = existing_parent.parent
         except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)